segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/SegmentStoreIntegrationTestBKZKHelper.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.host;
+
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import io.pravega.segmentstore.storage.impl.bookkeeper.BookKeeperConfig;
+import io.pravega.segmentstore.storage.impl.bookkeeper.BookKeeperServiceRunner;
+import io.pravega.test.common.TestUtils;
+import java.util.ArrayList;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import lombok.val;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+
+/**
+ * Helper class for Segment store integration tests.
+ * This class sets up the bookkeeper and zookeeper and sets up the correct values in config.
+ */
+public class SegmentStoreIntegrationTestBKZKHelper {
```

Comment:
1. Can we find a shorter and more appropriate name for this? It looks too long and clunky.
2. Make it implement `AutoCloseable`
3. Rename `setup` to `initialize` or `start` 
4. Rename `tearDown` to `close` 
5. Make `BOOKIE_COUNT` an argument.